.github/workflows/cd-pipeline.yml

```diff
@@ -79,3 +79,21 @@ jobs:
           git merge ${{ github.sha }} --no-squash -X theirs
           git push
 
+  deploy-to-staging:
+    needs: approval-to-deploy
```

Comment:
Should this be "merge-to-release"?  As written, it looks like it will deploy to Perf and Staging simultaneously.

Reply:
`merge-to-release` exists and is on line 58 (part of the existing code).